os/Kconfig

```diff
@@ -241,6 +241,15 @@ config AUTOGEN_MEMORY_LDSCRIPT
 
 endif # BUILD_PROTECTED
 
+config MPU_STACKGUARD
+	bool "Stack Overflow Protection"
+	depends on APP_BINARY_SEPARATION
+	depends on ARCH_USE_MPU
+	default y
+	---help---
+		Enables stack overflow protection.
+		It protects each task's stack overflow outside its max allocated stack memory.
```

Comment:
Please mention that this is applicable only to user mode tasks.

Reply:
done